charts/timed/Chart.yaml

```
@@ -2,8 +2,8 @@ apiVersion: v2
 name: timed
 description: Chart for Timed application
 type: application
-version: 0.6.3
-appVersion: v1.1.3
+version: 0.6.4
```

With regards to the Helm chart configuration this would need to be a minor version bump, because we're removing configuration options and replacing them with new ones.
```suggestion
version: 0.7.0
```

---

done